From 3e344ab44695f49dc25fd05ad4fa26f1b4c779b0 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <baimingcong@uniontech.com>
Date: Fri, 21 Jun 2024 14:58:38 +0800
Subject: [PATCH 126/360] arch: loongarch: configs: disable DRM_LOONGSON by
 default

This module is currently not yet ready and causes boot failure on some
systems. Use SYSFB_SIMPLEFB and use the EFI Framebuffer to drive the display
more reliably.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
---
 arch/loongarch/configs/deepin_loongarch_desktop_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
index 561a55b17db6..36a7e79dc3ae 100644
--- a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
+++ b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
@@ -3593,7 +3593,6 @@ CONFIG_DRM_I2C_ADV7511=m
 CONFIG_DRM_I2C_ADV7511_AUDIO=y
 CONFIG_DRM_CDNS_DSI=m
 CONFIG_DRM_CDNS_MHDP8546=m
-CONFIG_DRM_LOONGSON=m
 CONFIG_DRM_ETNAVIV=m
 CONFIG_DRM_LOGICVC=m
 CONFIG_DRM_ARCPGU=m
-- 
2.20.1

